infrastructure/deploy.sh

```diff
@@ -9,30 +9,38 @@ php -version
 
 ### Write-out .htaccess
 
+
+Tristan  3:32 PM
 cat << '__EOF__' > $(System.DefaultWorkingDirectory)/$(Release.PrimaryArtifactSourceAlias)/.htaccess
```

Comment:
Thoughts on frontloading script with all the default azure pipeline specific stuff, so that this is closer to runnable in other environments (and possible to dedup between other scripts)?

For example:

```suggestion
WORKDIR=${WORKDIR:-$(System.DefaultWorkingDirectory)}
SOURCE_ALIAS=${SOURCE_ALIAS:-$(Release.PrimaryArtifactSourceAlias)}
cat << '__EOF__' > $WORKDIR/$SOURCE_ALIAS/.htaccess
```

likely also need a bang line at the top, like

```bash
#!/usr/bin/env bash
```

Reply:
`envsubst` (which [seems to be available on azure pipeline environment](https://github.com/NewFuture/DDNS/blob/914f3c09c3c133b1883d5002462d68cfe59984aa/azure-pipelines.yml#L64-L65)) might be helpful here to copy the existing htaccess from elsewhere in the repo, but values into a template. We could also use envsubst in docker build, from same authoritative file.

So it uses a template (prob not  actually something we'd store in `infrastructure/php-container/src/`) to generate the deployable one.

```bash
envsubst < $WORKDIR/$SOURCE_ALIAS/infrastructure/php-container/src/.htaccess > $WORKDIR/$SOURCE_ALIAS/.htaccess
```

See: https://stackoverflow.com/a/14157575

Reply:
I appreciate the comments, and I have a separate issue defined to make these scripts more useful in multiple environments. #1843 

This PR is really just a quick and dirty record of what is working for the dev server, and I don't want to mess with that at the moment 😆